pkg/controller/migration/convert/core.go

```diff
@@ -145,3 +216,21 @@ func handleFelixNodeMetrics(c *components, install *Installation) error {
 
 	return nil
 }
+
+func checkTolerations(existing []corev1.Toleration, expected ...corev1.Toleration) error {
+	if len(existing) != len(expected) {
+		return fmt.Errorf("missing expected tolerations. have: %+v. expecting: %+v", existing, expected)
+	}
+
+OUTER:
+	for _, t := range expected {
+		for _, k := range existing {
+			if k == t {
+				continue OUTER
```

Comment:
This is just to continue the outer loop so doing a break here should work.

Reply:
I don't think that's correct - If I break the inner loop, I'll immediately hit the `return fmt.Errorf`

Reply:
Yep you're right.
Do you like this pattern? It is just a GOTO which I always feel should be avoided if possible.
One alternative would be to use a bool for if the toleration is found and use that to skip the `return fmt.Errorf`, then break would work.
But if you're not bothered by the 'goto' I can live with it.